packages/material-ui/src/Avatar/Avatar.js

```diff
@@ -50,22 +50,22 @@ const Avatar = React.forwardRef(function Avatar(props, ref) {
     ...other
   } = props;
 
-  let children = null;
   const img = src || srcSet;
 
   if (img) {
     children = (
-      <img
-        alt={alt}
-        src={src}
-        srcSet={srcSet}
-        sizes={sizes}
-        className={classes.img}
-        {...imgProps}
-      />
+      <>
```

Comment:
For consistency, we don't this syntax in the codebase, at least, not to date. We can't apply a `key` to the element with it.

```suggestion
      <React.Fragment>
```


@mui-org/core-contributors any preference?

Reply:
Okay, no problem. I will change this.
There is something else that makes the code fail also

Reply:
Yes, it seems to be an issue with const vs let.